hw5/pkg/board.go

```diff
@@ -0,0 +1,88 @@
+package game
+
+import (
+	"fmt"
+	"os"
+)
+
+type Board struct {
+	CursorPosX int
+	CursorPosY int
+	Cells      [][]Cell
+}
+
+type Cell struct {
+	Value string
+	ID    int
+}
+
+func (cell *Cell) setValue(value string) {
+	cell.Value = value
+}
+
+func NewBoard(size int) *Board {
+	board := &Board{
+		Cells: make([][]Cell, size),
+	}
+	for i := 0; i < size; i++ {
+		board.Cells[i] = make([]Cell, size)
+		for k := 0; k < size; k++ {
+			board.Cells[i][k].Value = "_"
+		}
+	}
+	return board
+}
+
+func (board *Board) renderBoard(redraw bool, symbol string) {
+	if redraw {
+		fmt.Printf("\033[%dA", len(board.Cells))
+	}
+
+	for i, _ := range board.Cells {
+
+		for k, cell := range board.Cells[i] {
+			var printCell string
+			delimeter := " |"
+			if k == len(board.Cells[k])-1 {
+				delimeter = "\n"
+			}
+
+			if cell.Value != "_" && i == board.CursorPosX && k == board.CursorPosY {
+				printCell = ">" + cell.Value
+			} else if cell.Value == "_" && i == board.CursorPosX && k == board.CursorPosY {
+				printCell = ">" + symbol
+			} else {
+				printCell = " " + cell.Value
+			}
+
+			fmt.Printf(" %s%s", printCell, delimeter)
+		}
+	}
+}
+
+func (board *Board) makeTurn(symbol string) {
+
+	for {
+		keyCode := getInput()
+		if keyCode == escape {
```

Comment:
It would be more readable if you will use "switch-case"

Reply:
I will keep it in mind next time.